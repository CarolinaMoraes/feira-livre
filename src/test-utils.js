import React from 'react'
import ThemeProvider from "styles/ThemeProvider";
import { MemoryRouter } from 'react-router-dom';
import { render } from "@testing-library/react";

const TestProvider = ({ children }) => (
  <MemoryRouter>
    <ThemeProvider>
      {children}
    </ThemeProvider>
  </MemoryRouter>
)


export * from "@testing-library/react";

const customRender = (ui, options) => render(ui, { wrapper: TestProvider, ...options });

export { customRender as render };
