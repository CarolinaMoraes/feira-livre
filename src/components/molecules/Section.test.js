import React from "react";
import { render } from "test-utils";

import Section from "./Section";


test('it renders correctly', () => {
  const { asFragment } = render(<Section></Section>);

  expect(asFragment()).toMatchSnapshot();
});
