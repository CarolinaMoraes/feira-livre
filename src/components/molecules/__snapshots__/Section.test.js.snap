// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<DocumentFragment>
  .c1 {
  width: 100%;
  padding: 0 8px;
}

.c0 {
  padding: 40px 0;
}

.c0 h2 {
  margin-top: 0;
}

@media (min-width:576px) {
  .c1 {
    padding: 0 16px;
  }
}

@media (min-width:1200px) {
  .c1 {
    max-width: 1140px;
    margin: 0 auto;
    padding: 0;
  }
}

<div
    class="c0"
  >
    <div
      class="c1"
    />
  </div>
</DocumentFragment>
`;
